latch/ldata/path.py

```diff
@@ -0,0 +1,272 @@
+        match = node_id_regex.match(self.path)
+        if match:
+            return match.group("id")
+
+        if self._cache.node_id is None or self._cache.path != self.path:
+            if not load_if_missing:
+                return None
+            self.load()
+        return self._cache.node_id
+
+    def name(self, *, load_if_missing: bool = True) -> str:
+        if self._cache.name is None or self._cache.path != self.path:
+            if not load_if_missing:
+                return None
+            self.load()
+        return self._cache.name
+
+    def type(self, *, load_if_missing: bool = True) -> LDataNodeType:
+        if self._cache.type is None or self._cache.path != self.path:
+            if not load_if_missing:
+                return None
+            self.load()
+        return self._cache.type
+
+    def size(self, *, load_if_missing: bool = True) -> float:
+        if self._cache.size is None or self._cache.path != self.path:
+            if not load_if_missing:
+                return None
+            self.load()
+        return self._cache.size
+
+    def content_type(self, *, load_if_missing: bool = True) -> str:
+        if self._cache.content_type is None or self._cache.path != self.path:
+            if not load_if_missing:
+                return None
+            self.load()
+        return self._cache.content_type
+
+    def is_dir(self) -> bool:
+        return self.type() in dir_types
+
+    def iterdir(self) -> Generator["LPath", None, None]:
+        data = execute(
+            gql.gql("""
+            query LDataChildren($argPath: String!) {
+                ldataResolvePathData(argPath: $argPath) {
+                    finalLinkTarget {
+                        type
+                        childLdataTreeEdges(filter: { child: { removed: { equalTo: false } } }) {
+                            nodes {
+                                child {
+                                    name
+                                }
+                            }
+                        }
+                    }
+                }
+            }"""),
+            {"argPath": self.path},
+        )["ldataResolvePathData"]
+
+        if data is None:
+            raise FileNotFoundError(f"No such Latch file or directory: {self.path}")
+        if data["finalLinkTarget"]["type"].lower() not in dir_types:
+            raise ValueError(f"{self.path} is not a directory")
+
+        for node in data["finalLinkTarget"]["childLdataTreeEdges"]["nodes"]:
+            yield LPath(urljoins(self.path, node["child"]["name"]))
+
+    def rmr(self) -> None:
+        execute(
+            gql.gql("""
+            mutation LDataRmr($nodeId: BigInt!) {
+                ldataRmr(input: { argNodeId: $nodeId }) {
+                    clientMutationId
+                }
+            }
+            """),
+            {"nodeId": self.node_id},
+        )
+
+    def copy(self, dst: "LPath", *, verbose: bool = False) -> None:
```

Comment:
```suggestion
    def copy_to(self, dst: "LPath", *, verbose: bool = False) -> None:
```

ik its not semantic but imo makes it way more clear